test/rules/no-consecutive-blank-lines/test.ts.lint

```diff
@@ -14,20 +14,20 @@ class Clazz { // comment
 
 ~nil
         console.log("test");
-~nil                         [consecutive blank lines are disallowed]
+~nil [consecutive blank lines are disallowed]
 
     }
 
 
 ~nil
 }
-~nil  [consecutive blank lines are disallowed]
+~nil [consecutive blank lines are disallowed]
 
 //Begin whitespace
 // The next two lines of "code" contain only tabs or spaces, they are also considered "blank" lines
-~nil                         [consecutive blank lines are disallowed]
         
         
+~ [consecutive blank lines are disallowed]
```

Comment:
Alright, as long as it works I guess.  I've worked a bit with the microsoft-tslint-conrib repo and I find their test files _much_ easier to read and understand.  They can either write TS code as string inputs or refer to real pass/fail files.  Then they just compare the test result objects which contain the rule failure, with a line/char position.

[Here's an example of that with string input](https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/tests/NoBannedTermsTests.ts#L16-L27)
And [here's an example test](https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/tests/NoOctalLiteralTests.ts#L37-L147) that refers to [a real TS file](https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/test-data/NoOctalLiteralTestInput.ts)